client/src/features/visualizer-threejs/VisualizerInstance.tsx

```diff
@@ -6,7 +6,12 @@ import React, { useEffect, useRef } from "react";
 import { RouteComponentProps } from "react-router-dom";
 import * as THREE from "three";
 import { Box3 } from "three";
-import { ACCEPTED_BLOCK_COLORS, PENDING_BLOCK_COLOR, TIME_DIFF_COUNTER, ZOOM_DEFAULT } from "./constants";
+import {
+  ACCEPTED_BLOCK_COLORS,
+  PENDING_BLOCK_COLOR,
+  TIME_DIFF_COUNTER,
+  ZOOM_DEFAULT,
+} from './constants'
```

Comment:
It's fine this time, no need to fix, but please don't commit changes with only formatting modified. I suppose the editor did it or something :D, but this way we can have conflicts later for no real reason.